data_acquisition/data_acquisition.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import os
+import sys
+import psycopg2
+import getpass
+import logging
+
+# set up logging
+logging.basicConfig(format='%(asctime)s - %(levelname)s @' +
+                    '%(module)s-%(funcName)s : %(message)s',
+                    level=logging.INFO)
+
+# for connecting to remote database in the future
+# conn = psycopg2.connect(database='yourdb', user='dbuser', password='abcd1234', host='server', port='5432', sslmode='require')
+
+def request_ziprecruiter_jobs(page_num, api_key):
+  url = "https://api.ziprecruiter.com/jobs/v1?search=internship%20Job&days_ago=1&page={page_num}&jobs_per_page=20&api_key={api_key}" \
+  .format(page_num=page_num, api_key=api_key)
+  response = requests.get(url).json()
+  if response["success"]:
+    con = psycopg2.connect(dbname='postgres', user=getpass.getuser(), host='localhost', password='Pa55word')
+    cur = con.cursor()
+    # parse ZipRecruiter's JSON response
+    for job in response["jobs"]:
+      job_title = job["name"]
+      link = job["url"]
+      description = job["snippet"]
+      city = job["city"]
+      state = job["state"]
+      country = job["country"]
+      company_name = job["hiring_company"]["name"]
+      salary_min = job["salary_min"]
+      # add to database
+      cur.execute(
+        "INSERT INTO Job VALUES(DEFAULT,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s)", \
+        (job_title, link, description, city, state, country, 0.0, 0.0, company_name, None, salary_min) \
```

Comment:
Possibly a good idea to add the fields to the INSERT INTO statement as well because on my machine the order of the columns wasn't correct and this caused an error. This will probably not be a big deal though if we are moving to Docker the column IDs should be standardized.